src/libraries/shims/ApiCompat.proj

```diff
@@ -26,13 +14,27 @@
     <NetFxReference Include="System.DirectoryServices.Protocols" />
   </ItemGroup>
 
+  <Target Name="SetApiCompatFiles">
+    <PropertyGroup>
```

Comment:
Just to understand, does this have to be inside a target so that `TargetFrameworkIdentifier` and other properties that depend on `TFM`, are set at the right time instead of on the `PropertyGroup` evaluation step were they're not available yet?

Reply:
right, let me add a comment.